packages/grafana-ui/src/utils/standardEditors.tsx

```diff
@@ -230,6 +231,13 @@ export const getStandardOptionEditors = () => {
     editor: NumberValueEditor as any,
   };
 
+  const slider: StandardEditorsRegistryItem<number, number[]> = {
+    id: 'slider',
+    name: 'Slider',
+    description: 'Allows slider input returning an array of numeric min & max values',
```

Comment:
I think slider should return just a number, as number input does. min/max should be responsibility of range slider.

Reply:
I will adjust the code so wel get just retrieve a number value 👍